Planner/src/AddNewTask.java

```diff
@@ -0,0 +1,289 @@
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.JCheckBox;
+import javax.swing.JComboBox;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+public class AddNewTask implements ActionListener {
+	
+	DefaultComboBoxModel<User> usersList;
+	ArrayList<Task> tasks;
+	ArrayList<User> users;
+	DefaultComboBoxModel<Task> tasksToday;
+	
+	public AddNewTask(DefaultComboBoxModel<User> usersList, ArrayList<Task> tasks, ArrayList<User> users, DefaultComboBoxModel<Task> tasksToday) {
+		this.usersList= usersList;
+		this.tasks= tasks;
+		this.users= users;
+		this.tasksToday= tasksToday;
+	}
+	
+	// button for adding new task
+	@Override
+	public void actionPerformed(ActionEvent arg0) {
+
+		// lists with years, months, days, hours and minutes- JComboBoxes in dialog for adding a new task
+		DefaultComboBoxModel<String> dayListS = new DefaultComboBoxModel<>();
+		DefaultComboBoxModel<String> dayListE = new DefaultComboBoxModel<>();
+		JList listWithDayS = new JList(dayListS);
+		JList listWithDaysE = new JList(dayListE);
+		for (int i = 1; i <= 31; i++) {
+			dayListS.addElement(String.format("%02d", i));
+			dayListE.addElement(String.format("%02d", i));
+		}
+	
+		DefaultComboBoxModel<String> monthListS = new DefaultComboBoxModel<>();
+		DefaultComboBoxModel<String> monthListE = new DefaultComboBoxModel<>();
+		JList listWithMonthS = new JList(monthListS);
+		JList listWithMonthE = new JList(monthListE);
+		for (int i = 1; i <= 12; i++) {
+			monthListS.addElement(String.format("%02d", i));
+			monthListE.addElement(String.format("%02d", i));
+		}
+	
+		DefaultComboBoxModel<String> yearListS = new DefaultComboBoxModel<>();
+		DefaultComboBoxModel<String> yearListE = new DefaultComboBoxModel<>();
+		JList listWithYearS = new JList(yearListS);
+		JList listWithYearE = new JList(yearListE);
+		for (int i = 2018; i < 2040; i++) {
```

Comment:
Here the for should start at Calendar.getInstance().get(Calendar.YEAR), because otherwise when we are in 2019 it will start from 2018, but we don't want that, we want it always to start from current year and end at current year + 20 years lets say.